lib/profiling/drhook/drhook_psy.f90

```diff
@@ -36,7 +36,9 @@
 
```

Comment:
Yes (that was one reason why the update took that long, I couldn't get DrHook to work the same way it did on my previous VM - it only worked when using mpi, which I didn't need to do previously ... turned out in the past I had mpi installed system-wide, so the MPI lib was found when linking, now I was using the mpi from in the LFRic installation) *sigh* Short, it should be all good, and I assume this will never be a automated test (due to the license of DrHook),